api/src/models/nfl-players.model.ts

```diff
@@ -0,0 +1,54 @@
+import { Sequelize, DataTypes, Model, Optional } from 'sequelize';
+import { NFLPlayer } from '@/interfaces/nfl-players.interface';
+
+export type NFLPlayerCreationAttributes = Optional<NFLPlayer, 'id' | 'player_name' 
+  | 'teamId' | 'position' | 'player_portrait'>;
+
+export class NFLPLayerModel extends Model<NFLPlayer, NFLPlayerCreationAttributes> implements NFLPlayer {
+  public id: number; 
+  public player_name: string; 
+  public teamId: number;
+  public position: string; 
+  public player_portrait: string; 
+
+  public readonly createdAt!: Date;
+  public readonly updatedAt!: Date;
+}
+  
+export default function (sequelize: Sequelize): typeof NFLPLayerModel {
+  NFLPLayerModel.init(
+    {
+      id: {
+        autoIncrement: true,
+        primaryKey: true,
+        type: DataTypes.INTEGER,
+      }, 
+      player_name: {
+        allowNull: false,
+        unique: 'playerNameUnique', 
```

Comment:
If you want to use the unique tag for `player_name`, I think that you should create a composite key to put in the `player_name` and `player_portrait` unique fields. That way, the code will ensure that no two rows will have the same `player_name` AND `player_portrait`, but two rows may have the same `player_name` and different `player_portrait`

https://sequelize.org/docs/v6/core-concepts/model-basics/#column-options

Reply:
I think I will just remove the unique from player, I do not think we need a double check

Reply:
Now that I'm thinking about more, I'm think that we should include the composite key for the unique tags on `player_name` and `player_portrait`. The unique tag will ensure that we don't have duplicates of the same player in the database. Then, the double check will make it so that if there are two players with the same name (i.e., Josh Allen, Josh Allen), then it will still be allowed as long as they have different pictures.